pkg/services/featuremgmt/registry.go

```diff
@@ -1929,6 +1929,12 @@ var (
 			Owner:        grafanaFrontendSearchNavOrganise,
 			Expression:   "false",
 		},
+		{
+			Name:        "grafanaPathfinder",
+			Description: "Enables Pathfinder app",
+			Stage:       FeatureStagePrivatePreview,
```

Comment:
is this the correct stage? Usually new feature flags are added as `FeatureStageExperimental`

Reply:
That should be it, updated @andresmgot